.github/workflows/release-please.yaml

```diff
@@ -0,0 +1,63 @@
+---
+name: Release Please
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  release:
+    name: Process Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@cc61a07e2da466bebbc19b3a7dd01d6aecb20d1e
+        id: release
+        with:
+          command: manifest
```

Comment:
I'm using built in actions tokens instead of PATs. 

Built in tokens have a limitation that they don't trigger other actions if a bot performs the actions, which is why say when the bot opens a PR it does not end up triggering unit tests.

I didn't want to start using PATs because then it would be linked to one of our accounts so I figured that I'd live with the consequences and do things like have to manually run tests myself. 

If we run into any issues with this flow then I'll switch to using pats.

One thing I haven't figured out yet is how to deal with dependencies yet, e.g. bumping AS and then bumping Rails then all etc...